README.md

```diff
@@ -1,74 +1,50 @@
+# Quick Start
+## Deploy ob-operator
+### Using Helm
+[Helm](https://github.com/helm/helm) is a package management tool for Kubernetes, please refer to the helm documentation to install the helm client.
 
-ob-operator 现已支持 OceanBase 集群的创建、删除，完整的节点生命周期管理，并通过 Service 的形式暴露给用户使用。后续会支持租户管理、ob-proxy 管理、多 Kubernetes 集群等功能。
-
-目前支持通过 YAML、Kustomize 来部署 ob-operator。后续会支持 Helm 形式部署。
-
-![01](docs/img/01.png)
-
-## 环境依赖
-
-构建 ob-operator 需要 Go 1.16 版本及以上。
-
-运行 ob-operator 需要 Kubernetes 1.16 版本及以上。
-尚未支持多 Kubernetes 版本，后续会支持。
-
-## 支持的 OceanBase 版本
-
-后续会不断支持新的 OceanBase 社区版本。
-暂不支持 3.1.2 之前的旧的 OceanBase 版本。目前仅对 OceanBase 社区版 3.1.2 进行了测试。
-
-## 开始体验
-
-### 使用 YAML 部署 ob-operator
-
-1. 部署 CRD 相关文件。
-
-```yaml
-kubectl apply -f ./deploy/crd.yaml
 ```
-
-2. 部署 ob-operator 相关文件。
-
-注意配置 `--cluster-name` 启动参数，推荐与 Kubernetes 集群名称保持一致。
-
-```yaml
-kubectl apply -f ./deploy/operator.yaml
+helm repo add ob-operator https://oceanbase.github.io/ob-operator/
+helm install ob-operator ob-operator/ob-operator --namespace=oceanbase-system --create-namespace  --version=1.1.0
 ```
 
-3. 配置节点 label。
-
-需要将 Kubernetes 节点打 label，label 需要与 obcluster.yaml 中 `nodeSelector` 配置相匹配。
-ob-operator 会将 Pod 调度到具有相应 label 的节点上。
-
-推荐配置 label 的 key 为 `topology.kubernetes.io/zone`。
-
-```yaml
-kubectl label node nodename topology.kubernetes.io/zone=zonename
+### Using configuration file
+The configuration files are located under deploy directory, using the following commands to deploy ob-operator.
 ```
-
-4. 部署 OceanBase 集群。
-
-```yaml
-kubectl apply -f ./deploy/obcluster.yaml
+# Deploy CRD
+kubectl apply -f deploy/crd.yaml
+# Deploy ob-operator
+kubectl apply -f deploy/operator.yaml
 ```
 
-5. 集群部署成功后可以使用 Service 的 ClusterIP 连接 OceanBase 集群。
-
-## 文档
+## Deploy OceanBase cluster
+### Customize configuration file
+`deploy/obcluster.yaml` defines an OceanBase cluster, including deployment topology, resources, storages etc. You can configure your own OceanBase based on this file.
 
-参考 [ob-operator 文档](docs/hello-cn.md)。
 
-## 获取帮助
+### Label node
+Ob-operator distributes observer pods on selected nodes to achive high availablity, by matching specific labels, ob-operator knows where to locate an observer pod, the node labels must includes the labels configured under nodeSelector in `deploy/obcluster.yaml`.
+You can use the following command to label a node.
+```
+kubectl label node ${node_name} ${label_name}=${label_value}
+```
 
-如果您在使用 ob-operator 时遇到任何问题，欢迎通过以下方式寻求帮助：
+### Deploy OceanBase
+Using the following command to deploy OceanBase Cluster
+```
+kubectl apply -f deploy/obcluster.yaml
+```
 
-- [GitHub Issue](https://github.com/oceanbase/ob-operator/issues)
-- [官方网站](https://open.oceanbase.com/)
+### Connect to OceanBase Cluster
+After successfully deployed OceanBase cluster, you can connect to OceanBase cluster via any observer pod's ip or the service address created by ob-operator named `svc-${cluster_name}`
 
-## 许可证
+# Contributing
```

Comment:
Do you need a module that explains how to manage the database, and guide users to the corresponding link

Reply:
This should be a todo, there's a lot of contents to be added, including architecture of the project, code explanation, test guide etc.